### PR TITLE
created branch desktop-version

### PR DESCRIPTION
**Pull request by OybekKayumov**
This branch is for start create a desktop version:

Updated .html
Updated .html and .css
Added shape
Updated .html and .css
Updated .html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,207 +71,244 @@ <h2 class="media-h2">LET'S CONNECT</h2>
...
-                        <img class="card-image-tonic1" src="./images/works2.png" alt="works1">
-                        <h3 class="card-title-tonic1">Multi-Post <br> Stories</h3>
-                        <div class="canopy-frm-tonic1">
-                            <div class="client-tonic1">
-                                <h3>CANOPY</h3>
+            <div class="item">
+                <div class="card">
+                    <div class="card-body">
+                        <div class="card-image img-right">
+                            <img class="card-image" src="./images/desk-works2.png" alt="works1">
```
Comment (leolpaz) [inline index.html]: - [ ] When making a website responsive, it's very important that we do not break the previous layouts we created, unfortunately, the layout is breaking here, this class used to be called on your CSS: card-image-tonic1 and was replaced by card-image1, kindly fix the class here so the properties for width and height apply again

As this and other images mentioned below without the class are breaking the site on mobile:
image

File: index.html [hunk 2]
```diff
@@ -64,207 +71,244 @@ <h2 class="media-h2">LET'S CONNECT</h2>
...
-                        <img class="card-image-tonic1" src="./images/works3.png" alt="works1">
-                        <h3 class="card-title-tonic1">Tonic</h3>
-                        <div class="canopy-frm-tonic1">
-                            <div class="client-tonic1">
-                                <h3>CANOPY</h3>
+            <div class="item">
+                <div class="card">
+                    <div class="card-body">
+                        <div class="card-image">
+                            <img class="card-image img-right" src="./images/desk-works3.png" alt="works1">
```
Comment (leolpaz) [inline index.html]: - [ ] Kindly apply the same fix as mentioned above

File: index.html [hunk 3]
```diff
@@ -64,207 +71,244 @@ <h2 class="media-h2">LET'S CONNECT</h2>
...
-                        <img class="card-image-tonic1" src="./images/works4.png" alt="works1">
-                        <h3 class="card-title-tonic1">Multi-Post <br> Stories</h3>
-                        <div class="canopy-frm-tonic1">
-                            <div class="client-tonic1">
-                                <h3>CANOPY</h3>
+            <div class="item">
+                <div class="card">
+                    <div class="card-body">
+                        <div class="card-image img-right">
+                            <img class="card-image" src="./images/desk-works4.png" alt="works1">
```
Comment (leolpaz) [inline index.html]: - [ ] Kindly apply the same fix as mentioned above

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @OybekKayumov , 

You've done an amazing job so far, I'd like to highlight the following:
- Page looks clean on the desktop version
- No linter errors

One issue needs to be resolved before your approval, but you're almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (OybekKayumov) [reply index.html]: corrected, 
thanks for this comment
very useful.

File: index.html [hunk 4]
```diff
@@ -64,207 +71,244 @@ <h2 class="media-h2">LET'S CONNECT</h2>
...
-                            <div class="client-tonic1">
-                                <h3>CANOPY</h3>
+        
+        <div class="works-container">
+            
+            <div class="item">                
+                <div class="card">                    
+                    <div class="card-body">
+                        <div class="card-image-tonic1"> 
+                            <img class="card-image1-tonic1 img-right" src="./images/desk-works1.png" alt="works1">
```
Comment (rustam-tolipov) [inline index.html:82]: - [ ] [OPTIONAL] Everything is working well but there was a typo. Please kindly change  `card-image1-tonic1` to `card-image-tonic1`, I think you added an additional `1` by mistake 👍

**Review by rustam-tolipov — APPROVED** [review]
## Hi @OybekKayumov 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (OybekKayumov) [reply index.html:82]: Fixed,
Thank you.

**Pull request merged**